README.md

```diff
@@ -116,6 +116,17 @@ end
 This also allows you to define multiple state machines on the same model. Note that event
 and state names still have to be unique for the whole model.
 
+If you want to have the same state names for different state machines on the same model,
+you have to use the `prefix:` keyword argument.
+
+To use a prefix, pass it to the `.state_machine` method as a keyword argument:
```

Comment:
IMO this is overly specific towards the implementation and does not explain the need for a prefix well.

How about this?

```suggestion
If you define multiple state machines on the same model, and use state names more than once,
you must use a prefix to avoid name collisions.

A prefix may be specified by passing the `:prefix` option when declaring your state machine:
```